libmscore/chord.cpp

```diff
@@ -3098,7 +3098,36 @@ void Chord::sortNotes()
       std::sort(notes().begin(), notes().end(),
          [](const Note* a,const Note* b)->bool { return b->line() < a->line(); }
          );
-     }
+      }
+
+//---------------------------------------------------------
+//   nextTiedChord
+//    Return next chord if all notes in this chord are tied to it.
+//    Note: the next chord might have extra notes that are not tied
+//    back to this one. Set sameSize=true to return 0 in this case.
+//---------------------------------------------------------
+
+Chord* Chord::nextTiedChord(bool backwards, bool sameSize)
```

Comment:
It was supposed to mean "look backwards" (i.e. return the previous one instead of the next one). It worked when I was using `segment()->nextChordRest(track, backwards);`, but then I swapped that for `segment()->next1(Segment::Type::ChordRest);` and it looks like I forgot to handle the backwards case.